tests/inlineeditoruiview.js

```diff
@@ -91,6 +101,29 @@ describe( 'InlineEditorUIView', () => {
 			it( 'sets the default value of the #viewportTopOffset attribute', () => {
 				expect( view.viewportTopOffset ).to.equal( 0 );
 			} );
+
+			// Sometimes this test can fail, due to the fact that it have to wait for async ResizeObserver execution.
+			it( 'max-width should be set to the width of the editable element', done => {
+				const viewElement = view.editable.element;
+
+				global.document.body.appendChild( viewElement );
+				expect( global.document.body.contains( viewElement ) ).to.be.true;
+
+				viewElement.style.width = '400px';
+
+				// Unfortunately we have to wait for async ResizeObserver execution.
+				// ResizeObserver which has been called after changing width of viewElement,
+				// needs 2x requestAnimationFrame or timeout to update a layout.
+				// See more: https://twitter.com/paul_irish/status/912693347315150849/photo/1
+				setTimeout( () => {
+					const editableWidthWithPadding = toPx( new Rect( viewElement ).width );
+					expect( view.toolbar.maxWidth ).to.be.equal( editableWidthWithPadding );
+
+					global.document.body.removeChild( viewElement );
+
+					done();
+				}, 500 );
```

Comment:
Wouldn't 50ms be enough? 500 is quite a lot. 

Given the number of tests we have, if we let some of them slow the execution by .5s freely, soon we'll have to wait extra minutes for the tests to finish.

Reply:
No timeout needed anymore 👌(see https://github.com/ckeditor/ckeditor5-editor-inline/pull/65#discussion_r384446873)